### ISSUE TITLE
PLW3301: min/max false positive with iterables

### ISSUE THREAD

**Issue by lovetox** [charliermarsh/ruff#4682]
ruff: 0.0.267

```
tuples_list = [
    (1, 2),
    (2, 3),
    (3, 4),
    (4, 5),
    (5, 6),
]

max(max(tuples_list))
```

This raises `PLW3301 [*] Nested `max` calls can be flattened`

i don’t see how this can be flattend, if you only call max() once, it will return a tuple instead of the integer

**Comment by charliermarsh** [comment]
Thanks, you're right :)

**Comment by charliermarsh** [comment]
Interestingly, Pylint flags this too.
